packages/salesforcedx-vscode-core/src/context/workspaceOrgType.ts

```diff
@@ -28,12 +28,12 @@ export async function getWorkspaceOrgType(
       err.message
     )
   );
-  return isScratchOrg ? OrgType.SourceTracked : OrgType.NonSourceTracked;
+  return isScratchOrg ? OrgType.ScratchOrg : OrgType.NonScratchOrg;
```

Comment:
Sandboxes can be source-tracked and are not accounted for here.  This change makes the Enum more accurate.

Reply:
I rolled this back at some point, because ideally we want this to be accurate as `SourceTracked` and `NonSourceTracked`.  

I updated the function in [this commit](https://github.com/forcedotcom/salesforcedx-vscode/pull/4597/commits/78f4af382eaad370d70226e5fb854dc956c9640c) to use core to determine if the org is source-tracked.  Core's [tracksSource() function](https://github.com/forcedotcom/sfdx-core/blob/main/src/org/org.ts#L574) takes into account Sandboxes and will return true if the org is a scratch or or a source-tracked sandbox.

Reply:
FYI this comment is outdated.  See discussions below about deciding to use org.supportsSourceTracking().